src/chainparams.cpp

```diff
@@ -42,7 +42,6 @@ class CMainParams : public CChainParams {
         consensus.nMajorityEnforceBlockUpgrade = 750;
         consensus.nMajorityRejectBlockOutdated = 950;
         consensus.nMajorityWindow = 4000;
-        // TODO generate harder genesis block
```

Comment:
Right, but if for any reason we didn't do that, the current difficulty is sufficient so the comment is no longer needed, IMHO.